chartfx-chart/src/main/java/de/gsi/chart/Chart.java

```diff
@@ -273,40 +277,53 @@ protected void invalidated() {
         }
     };
 
-    private final StyleableObjectProperty<Side> toolBarSide = CSS.createObjectProperty(this, "toolBarSide", Side.TOP, false, StyleConverter.getEnumConverter(Side.class), (oldVal, newVal) -> {
-        AssertUtils.notNull("Side must not be null", newVal);
-        // remove tool bar from potential other chart side pane locations
-        Chart.this.setTop(null);
-        Chart.this.setBottom(null);
-        Chart.this.setLeft(null);
-        Chart.this.setRight(null);
-        switch (newVal) {
-        case LEFT:
-            getToolBar().setOrientation(Orientation.VERTICAL);
-            Chart.this.setLeft(getToolBar());
-            break;
-        case RIGHT:
-            getToolBar().setOrientation(Orientation.VERTICAL);
-            Chart.this.setRight(getToolBar());
-            break;
-        case BOTTOM:
-            getToolBar().setOrientation(Orientation.HORIZONTAL);
-            Chart.this.setBottom(getToolBar());
-            break;
-        case TOP:
-        default:
-            getToolBar().setOrientation(Orientation.HORIZONTAL);
-            Chart.this.setTop(getToolBar());
-            break;
-        }
-        return (newVal);
-    }, this::requestLayout);
+    private final StyleableObjectProperty<Side> toolBarSide = CSS.createObjectProperty(this, "toolBarSide", Side.TOP, false,
+            StyleConverter.getEnumConverter(Side.class), (oldVal, newVal) -> {
+                AssertUtils.notNull("Side must not be null", newVal);
+                // remove tool bar from potential other chart side pane locations
+                Chart.this.setTop(null);
+                Chart.this.setBottom(null);
+                Chart.this.setLeft(null);
+                Chart.this.setRight(null);
+                switch (newVal) {
+                case LEFT:
+                    getToolBar().setOrientation(Orientation.VERTICAL);
+                    Chart.this.setLeft(getToolBar());
+                    break;
+                case RIGHT:
+                    getToolBar().setOrientation(Orientation.VERTICAL);
+                    Chart.this.setRight(getToolBar());
+                    break;
+                case BOTTOM:
+                    getToolBar().setOrientation(Orientation.HORIZONTAL);
+                    Chart.this.setBottom(getToolBar());
+                    break;
+                case TOP:
+                default:
+                    getToolBar().setOrientation(Orientation.HORIZONTAL);
+                    Chart.this.setTop(getToolBar());
+                    break;
+                }
+                return (newVal);
+            }, this::requestLayout);
 
     /**
      * Creates a new default Chart instance.
+     * 
+     * @param axes axes to be added to the chart
      */
-    public Chart() {
-        super();
+    public Chart(Axis... axes) {
```

Comment:
Codacy found an issue: [The constructor 'Chart(Axis...)' has an NPath complexity of 3600, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)